numpy/core/_add_newdocs.py

```diff
@@ -2702,6 +2702,12 @@
     -----
     Does not include memory consumed by non-element attributes of the
     array object.
+    
+    See Also
+    --------
+    sys.getsizeof
+        Memoery consumed by the object itself
+        without parents in case view
```

Comment:
Worth mentioning that in contrast with the comment above, this _does_ include memory consumed by non-element attributes.